client/src/components/controls/Timeline.vue

```diff
@@ -60,52 +60,51 @@ export default {
       this.update();
     },
     handLeftPosition(value) {
-      this.$refs.hand.style.left = (value ? value : "-10") + "px";
+      this.$refs.hand.style.left = `${value || '-10'}px`;
     },
     frame(frame) {
       if (frame > this.endFrame) {
         this.endFrame = Math.min(frame + 200, this.maxFrame);
       } else if (frame < this.startFrame) {
         this.startFrame = Math.max(frame - 100, 0);
       }
-    }
+    },
   },
   created() {
     this.update = throttle(this.update, 30);
   },
   mounted() {
-    var width = this.$refs.workarea.clientWidth;
-    var height = this.$refs.workarea.clientHeight;
-    var scale = d3
+    const width = this.$refs.workarea.clientWidth;
+    const height = this.$refs.workarea.clientHeight;
+    const scale = d3
       .scaleLinear()
       .domain([0, this.maxFrame])
       .range([0, width]);
     this.timelineScale = scale;
-    var axis = d3
+    const axis = d3
       .axisTop()
       .scale(scale)
       .tickSize(height - 30)
       .tickSizeOuter(0);
     this.axis = axis;
     this.g = d3
       .select(this.$refs.workarea)
-      .append("svg")
```

Comment:
`this.axis` - private non-reactive variable.  Should it be initialized in `created()` instead of first being introduced here?  I understand keeping it here because there isn't a really good initialization value other than `null`, just wondering opinions on that.

Reply:
Out of scope of this PR maybe.  Let's go through and clean up individual files in the future.